plugins/com.google.cloud.tools.eclipse.sdk/src/com/google/cloud/tools/eclipse/sdk/internal/Activator.java

```diff
@@ -37,6 +37,7 @@ public void preferenceChange(PreferenceChangeEvent event) {
   @Override
   public void start(BundleContext context) throws Exception {
     PreferenceConstants.getPreferenceNode().addPreferenceChangeListener(listener);
+    PreferenceConstants.initializeDefaults();
```

Comment:
It seems odd for a constants class to have an `initializeDefaults()` method.  It also seems odd to need an `initializeDefaults()` since `ScopedPreferenceStore` should be consulting the `DefaultScope` too.  We should be able to use the `org.eclipse.equinox.preferences.preferences` extension point to provide an initializer. This has the benefit of not actually changing the preference values unnecessarily.